assignments/lambda-classes.js

```diff
@@ -1 +1,134 @@
 // CODE here for your Lambda Classes
+
+class Person{
+    constructor(personAttributes){  
+    this.name = personAttributes.name;  
+    this.age = personAttributes.age;  
+    this.location = personAttributes.location  
+    }
+  
+    speak(){  
+     return `Hello, my name is ${this.name}, I am from ${this.location}.`;  
+     }  
+  }  
+     
+  class Instructor extends Person{  
+    constructor(instructorAttributes){  
+    super(instructorAttributes);  
+    this.specialty = instructorAttributes.specialty;  
+    this.favLanguage = instructorAttributes.favLanguage;  
+    this.catchPhrase = instructorAttributes.catchPhrase;  
+    
+      }
+     
+      demo(subject) {  
+   return `Today we are learning about ${subject}`;  
+    }
+     
+      grade(student,subject){  
+    return `${student.name} receives a perfect score on ${subject}`  
+     }
+    }
+  
+     
+  class Student extends Person{  
+    constructor(studentAttributes){  
+    super(studentAttributes);  
+    this.previousBackground = studentAttributes.previousBackground;  
+    this.className = studentAttributes.className;  
+    this.favSubjects = studentAttributes.favSubjects;  
+     }  
+    listsSubjects() {  
+    return `${this.favSubjects}`;  
```

Comment:
Since `favSubjects` is plural, you could assign it as an array here, then loop over it line for line.